viewer/vueapp/src/components/connections/Connections.vue

```diff
@@ -134,24 +134,6 @@
           </select>
         </div> <!-- /weight select -->
 
-        <!-- unlock button-->
```

Comment:
these buttons were moved into the overlay to save space on the tool bar

![image](https://user-images.githubusercontent.com/13872653/76992359-bed14380-6910-11ea-8fd6-d68826d04f6b.png)